serializations.md

```diff
@@ -644,6 +646,22 @@ The payload is a serialized signed channel off-chain transaction and can not be
 ]
 ```
 
+version 2:
```

Comment:
If I read this section, it looks both versions 1 and 2 are legal in Roma and Minerva and Fortuna.